README.md

```diff
@@ -69,6 +77,16 @@ an Appengine application deployed.
 
 Deploy the application using `gcloud app deploy`.
 
+If EHR writeback is enabled, a serverless connector must be created for the app
+to access resources in a VPC network where the EHR server is connected to. Then
+in the app.yaml file, add the following section:
```

Comment:
Which app.yaml file is this referring to?

Reply:
Added "in the app folder" statement.

Reply:
What app folder? You mean the app.yaml file in the root directory of this repo?

Can you link to that file? Can we add a commented out section that the file?

Reply:
Done.